packages/epo-widget-lib/src/atomic/Blinker/Blinker.test.tsx

```diff
@@ -15,38 +15,31 @@ const testProps = {
 
 describe("Blinker", () => {
```

Comment:
rearranged some of the tests in the Blinker after adding in the load callback from images